envoy/Makefile

```diff
@@ -74,10 +74,10 @@ GO_TARGETS= $(ENVOY_API_TARGETS) $(CILIUM_TARGETS) $(CILIUM_API_TARGETS) $(FILTE
 
 # Dockerfile builds require special options
 ifdef PKG_BUILD
-BAZEL_BUILD_OPTS = --spawn_strategy=standalone --genrule_strategy=standalone
+BAZEL_BUILD_OPTS = --experimental_strict_action_env --spawn_strategy=standalone --genrule_strategy=standalone
```

Comment:
I would not add this for the PKG_BUILD, as it currently works as intended. (when used for docker builds)